schemas/records.json

```diff
@@ -647,33 +519,39 @@
       "type": "object",
       "properties": {
         "rel": {
-          "type": "string",
-          "description": "The type or semantics of the relation."
+          "title": "Relation",
+          "type": "string"
         },
         "type": {
           "type": "string",
-          "description": "A hint indicating what the media type of the result of dereferencing the link should be."
+          "title": "Media Type"
         },
         "hreflang": {
           "type": "string",
-          "description": "A hint indicating what the language of the result of dereferencing the link should be."
+          "description": "A hint indicating what the language of the result of dereferencing the link should be.",
+          "options": {
+            "hidden": true
+          }
         },
         "title": {
           "type": "string",
-          "description": "Used to label the destination of a link such that it can be used as a human-readable identifier."
+          "title": "Title"
         },
         "length": {
-          "type": "integer"
+          "type": "integer",
+          "options": {
+            "hidden": true
+          }
         },
         "created": {
-          "type": "string",
-          "description": "Date of creation of the resource pointed to by the link.",
-          "format": "date-time"
+          "options": {
+            "hidden": true
+          }
         },
         "updated": {
-          "type": "string",
-          "description": "Most recent date on which the resource pointed to by the link was changed.",
-          "format": "date-time"
+          "options": {
+            "hidden": true
+          }
```

Comment:
This removed the validation, why?

Reply:
The plan is to auto-fill this field on file commit, so the user doesn't have to set it manually. Does options.hidden disable it in validation? That's new to me, so maybe also other fields mustn't be hidden?

My goal is to reduce the number of shown fields in order to reduce overload for a user editing this. If we require optional fields to be added in future (plus custom fields as I understand), we'd probably need to revert all the options.hidden fields anyway...

Reply:
You removed the data type and format, too. So it's not validated any longer.
Just adding the options is fine, but not override the whole schema.

Reply:
See above